assets/js/admin/admin.js

```diff
@@ -221,4 +221,60 @@
 		$(this).append(video);
 
 	});
+
+	// Change span with file name when user selects a file
+	$('#everest-forms-import').change(function(e){
```

Comment:
Please make sure every statement follows space after opening and closing of small brackets as per coding standards. I will not list comment for others below but make sure for others too xD

Reply:
Resolved with 483e8ff

Reply:
The comment has been addressed.